templates/zerver/help/import-from-slack.md

```diff
@@ -74,6 +74,9 @@ First
 skipping "Step 3: Create a Zulip organization, and log in" (you'll
 create your Zulip organization via the data import tool instead).
 
+Copy the `slack_data.zip` file you obtained above onto the Zulip
+server, and put it in `~zulip/deployments/current/`.
```

Comment:
Should we tell you to put in in /tmp/ and then just access it from /tmp/ below? 

I don't think we actually want to encourage putting random files in your deployment directory.

(Though I guess the `--output` option has the same issue).

Reply:
Sure.  Repushed.